doc/cli.rst

```diff
@@ -45,11 +45,62 @@ This again generates a Tensorboard event file ``evaluation/events.out.tfevents.*
 Hyperparameters
 ---------------
 
-The hyperparameters of the training and the wave function ansatz are specified through hydra config files. Predefined ansatzes can be found in ``.../deepqmc/src/deepqmc/conf/ansatzes``. The hyperparameters of a default model can be overwritten at the command line::
+In the following the most relevant settings for running experiments with DeepQMC are discussed.
 
-    $ deepqmc task.restdir=workdir ansatz=paulinet ansatz.omni_factory.gnn_factory.n_interactions=2
+Task
+____
 
-For convenience the configuration of the default ansatz is reproduced here:
+DeepQMC provides the above mentioned conigurations for the ``train``, ``evaluate`` and ``restart`` task. In order to override default hyperparameters of the experimental setup, such as the ``sample_size`` or the number of training ``steps`` or ``pretrain_steps``, hydra provides a simple syntax::
 
-.. literalinclude:: defaults.yaml
+        $ deepqmc task=train task.sample_size=2048 task.steps=50_000 task.pretrain_steps
```

Comment:
I don't think the `pretrain_steps` argument is correct.

1. It's missing a value
2. I think it needs the `+` sign in front.

Reply:
Thanks, yes something got messed up there.

Reply:
I've fixed this argument in my last commit. I used `+task.pretrain_steps=1000`